packages/form-js-editor/assets/form-js-editor-base.css

```
@@ -265,6 +267,10 @@
   color: var(--color-context-pad-item-hover-fill);
 }
 
+.fjs-editor-container .fjs-context-pad-item:focus-visible {
+  outline-color: var(--color-children-selected-border);
```

For me on Firefox on Linux I can only see the outline on the correct color if I do it like below. Is it the same on Firefox on Mac?
```suggestion
  outline: 2px solid var(--color-children-selected-border);
  border-radius: 2px;
```